internal/extractors/extract_data_test.go

```diff
@@ -12,30 +12,30 @@ import (
 func Example() {
 
 	data := []byte{
-		//1
+		//1 A continuation packet (started in previous file), should error in aggregator
 		0x90, 0xeb, 0x58, 0x00, 0x79, 0xd8, 0x00, 0x00, 0xdc, 0xad, 0xc8, 0x04, 0xda,
 		0x1c, 0x00, 0x00, 0x00, 0x00, 0x2e, 0x02, 0x00, 0x00, 0x64, 0x00, 0x00, 0x00, 0x00, 0xcc, 0xcc,
 		0xcc, 0xcc, 0x00, 0x08, 0x64, 0x88, 0x97, 0x00, 0x41, 0x10, 0x80, 0x19, 0x00, 0x00, 0x12, 0x19,
 		0xda, 0x7e, 0x00, 0x17, 0x01, 0x2f, 0x01, 0x31, 0x01, 0x2f, 0x01, 0x2f, 0x01, 0x2f, 0x01, 0x31,
 		0x01, 0x2e, 0x01, 0x32, 0x01, 0x30, 0x01, 0x2f, 0x01, 0x2f, 0x01, 0x2f, 0x01, 0x2d, 0x01, 0x30,
 		0x01, 0x2e, 0x01, 0x30, 0x01, 0x2b, 0x01, 0x2f, 0x01, 0x31, 0x01, 0x32, 0x01, 0x32, 0x01, 0x33,
 		0x01, 0x2e, 0x01, 0x31, 0x01, 0x2d, 0x01, 0x2e, 0x01, 0x20, 0xf7,
-		//2
+		//2 A STAT package should parse all the way through
 		0x90, 0xeb, 0x48, 0x00, 0x79,
 		0xd8, 0x00, 0x00, 0xfb, 0xad, 0xc8, 0x04, 0xda, 0x1c, 0x00, 0x00, 0x00, 0x00, 0x2e, 0x02, 0x00,
 		0x00, 0x64, 0x00, 0x00, 0x00, 0x00, 0xcc, 0xcc, 0xcc, 0xcc, 0x00, 0x08, 0x64, 0xc8, 0x98, 0x00,
 		0x31, 0x10, 0x03, 0x19, 0x00, 0x00, 0x12, 0x19, 0xe3, 0x39, 0x00, 0x01, 0x7f, 0x04, 0x02, 0x82,
 		0x04, 0x02, 0x02, 0x06, 0x01, 0x19, 0x12, 0x00, 0x00, 0x0c, 0xe3, 0x02, 0x00, 0x00, 0x00, 0x00,
 		0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x41, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
 		0x00, 0x30, 0xfb,
-		//3 Too short buffer
-		0x90, 0xeb, 0x48, 0x00, 0x79, 0xd8, 0x00, 0x00, 0xd9, 0xb0, 0xc8, 0x04, 0xda,
-		0x1c, 0x00, 0x00, 0x00, 0x00, 0x2e, 0x02, 0x00, 0x00, 0x64, 0x00, 0x00, 0x00, 0x00, 0xcc, 0xcc,
+		//3 Too short buffer (innosat tm header length too short for a STAT) and also bad CRC
+		// Package length for ramses is correct (3rd & 4th byte here 0x46 = 70, and the 16 bytes header)
+		0x90, 0xeb, 0x46, 0x00, 0x79, 0xd8, 0x00, 0x00, 0xd9, 0xb0, 0xc8, 0x04, 0xda, 0x1c, 0x00, 0x00, 0x00, 0x00, 0x2e, 0x02, 0x00, 0x00, 0x64, 0x00, 0x00, 0x00, 0x00, 0xcc, 0xcc,
```

Comment:
why remove newline here..?

Reply:
Oh that's not how I wanted it. The consistent way would have been to have 16 bytes for the first two rows since thats the length of the ramses and ramses tm headers.

Reply:
I made all 8 long and put comments showing Ramses Header,  Ramses TM Header, Ramses TM Payload.